core/components/atoms/metricInput/MetricInput.tsx

```diff
@@ -201,6 +201,11 @@ export const MetricInput = React.forwardRef<HTMLInputElement, MetricInputProps>(
 
   const onArrowClick = (e: React.MouseEvent<HTMLElement, MouseEvent>, direction: string) => {
     let newValue = Number(value || 0);
+    /**
+     * we are calculating the number of digits after decimal,
+     * to use it for rounding off the newValue post +-1
+     */
```

Comment:
Please update this message to `To calculate number of digits after decimal`

Reply:
Done